src/pages/orderSuccess.js

```diff
@@ -338,7 +338,7 @@ class OrderSuccessPage extends React.Component {
     const { collective, fromCollective, totalAmount, interval, currency, tier } = order;
     const referralOpts = objectToQueryString({ referral: fromCollective.id });
     const websiteUrl = process.env.WEBSITE_URL || 'https://opencollective.com';
-    const referralURL = `${websiteUrl}${fromCollective.path}${referralOpts}`;
```

Comment:
`path` might be better than slug because it's accommodating events(`/${parent.slug}/events/${this.slug}`).